src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx

```diff
@@ -1,56 +1,60 @@
 import React, { useState } from 'react';
-import TextInput from '@department-of-veterans-affairs/component-library/TextInput';
+import PropTypes from 'prop-types';
+import {
+  VaTextInput,
+  VaButton,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
```

Comment:
I'm pretty sure that we don't need a React binding for `VaTextInput` - native event handlers like `onInput` work just fine without the binding:

https://github.com/department-of-veterans-affairs/vets-website/blob/eac1ae10b3c9670b765ce2aea6e5546cc694c62c/src/applications/financial-status-report/components/PreSubmitSignature.jsx#L142-L147

`onBlur` and the `onClick` for the button should work the same way.

Reply:
Oh, good to know, thanks @bkjohnson!